tools/runners/run-unittests.sh

```diff
@@ -63,15 +63,21 @@ passed=0
 
 UNITTEST_TEMP=`mktemp unittest-out.XXXXXXXXXX`
 
+TERM_EMPTY="\033[0m"
+TERM_RED="\033[1;31m"
+TERM_GREEN="\033[1;32m"
+
 for unit_test in $UNITTESTS
 do
     cmd_line="$RUNTIME ${unit_test#$ROOT_DIR}"
     $RUNTIME $unit_test &>$UNITTEST_TEMP
     status_code=$?
 
+    testfile=$(basename $cmd_line)
```

Comment:
basename'ing the whole `$cmd_line` looks very strange. Isn't it the `$unit_test` that's being looked for?

Reply:
Yes, same as above in one of your comments, I didn't see the `$unit_test` variable. But I think it's unnecessary to display the absolute path to the file. How about only the filename or a relative path with something like this?
```testfile=$(realpath --relative-to . $unit_test)```

Reply:
realpath may also be an approach but there is already a way for shortening the filename. So, again, either `$(basename $unit_test)` or `${unit_test#$ROOT_DIR}`.